src/components/decks/DeckListing.vue

```diff
@@ -46,28 +73,32 @@ export default {
     showMine: {
       type: Boolean,
       default: false,
+    },
+    showForUser: {
+      type: Number,
```

Comment:
This needs to be a string; IsaacBot is actually unusual in that it's a completely numeric badge, but most of them have alphabet or punctuation characters in them, as well.